src/styles/_start_menu.scss

```diff
@@ -304,6 +295,24 @@
 
       .startMenuSearch {
         margin-top: 10px;
+        display: flex;
+        justify-content: space-between;
+
+        form {
+          width: 90%;
+        }
+
+        // fix select2 placeholder cutoff
+        // https://github.com/select2/select2/issues/291
+        .select2-search__field {
+          width: auto !important;
```

Comment:
yes I am, but the width of the textbox of select2 in which the user actually types somehow gets a width of 100px, which doesn't fit the placeholders